common/src/main/java/org/apache/nemo/common/ir/executionproperty/ExecutionProperty.java

```diff
@@ -42,6 +42,13 @@ public final T getValue() {
     return this.value;
   }
 
+  /**
+   * @return the class of the value type.
+   */
+  public final Class<? extends Serializable> getValueClass() {
```

Comment:
Why do you add this function? Can't we extract this class by `.getValue().getClass()`?

Reply:
You're right. It's been simple after all I'll get rid of the method.